s3_sync/s3_sync.py

```diff
@@ -0,0 +1,158 @@
+args = parser.parse_args()
+settings_file = args.settings_file
+
+# Read settings_file
+config = ConfigParser(interpolation = ExtendedInterpolation())
+config.read(settings_file)
+
+# Configure logbook logging
+logger = RotatingFileHandler(config['Logs']['logfile'], max_size = int(config['Logs']['max_size']),
+	backup_count = int(config['Logs']['backup_count']),
+	level = int(config['Logs']['level']))
+logger.push_application()
+logger = Logger(__name__)
+
+# Get Computer name
+computer_name = os.environ['COMPUTERNAME']
+
+def verify_settings_file_exists(settings_file):
+	if os.path.exists(settings_file):
+		return
+	else:
+		logger.error("Settings file not found:", settings_file)
+		sys.exit("[-] Settings file not found")
+
+def set_environment_variables():
+	os.environ['AWS_ACCESS_KEY_ID'] = config['AWS']['AWS_ACCESS_KEY_ID']
+	os.environ['AWS_SECRET_ACCESS_KEY'] = config['AWS']['AWS_SECRET_ACCESS_KEY']
+	os.environ['slack_webhook_url'] = config['Slack']['webhook_url']
+
+def verify_local_folder_exists(local_video_records_done_folder):
+	"""
+    Check whether the local video records done folder exists
+
+    Args:
+      local_video_records_done_folder (str): The path of the local video
+      	records done folder.
+
+    Returns:
+      If folder exists return None, and if not, logs error and exit.
+	"""
+	if os.path.exists(local_video_records_done_folder):
+		return
+	else:
+		logger.exception("Local Video Records Done folder not found")
+		sys.exit("[-] Local Video Records Done folder not found")
+
+def verify_aws_cli_installed(aws_cli_binary):
+	"""
+	Check whether AWS CLI is installed
+
+	Args:
+	  aws cli binary (str): absolute path to aws cli binary file.
+
+	Returns:
+	  If file exists, return None, else log error and exit.
+
+	"""
+	if os.path.exists(aws_cli_binary):
+		return
+	else:
+		logger.exception("Could not find AWS CLI executable:")
+		sys.exit("[-] Could not find AWS CLI executable")
+
+def verify_s3_bucket_exists(s3_bucket_name):
+	"""
+	Check whether S3 bucket exists
+
+	Args:
+	  s3_bucket_name (str): The s3 bucket name
+
+	Returns:
+	  list: if connection established and bucket found, return list of
+	  	objects in bucket otherwise error and exit on any issues trying
+	  	to list objects in bucket.
+	"""
+	try:
+		ls_s3_bucket_cmd = 'aws s3 ls {}'.format(s3_bucket_name)
+		subprocess.run(ls_s3_bucket_cmd, shell=True, check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+		return
+	except subprocess.SubprocessError:
+		logger.exception("Failed to list specified s3 bucket")
+		sys.exit("[-] Failed to list specified s3 bucket")
+
+def notify_slack_channel(slack_message):
+	"""
+	Send notification to Slack Channel
+
+	Args:
+	  slack_message (str): message to send to slack
+	"""
+	try:
+		resp = requests.post(os.environ.get('slack_webhook_url'), json={"text": slack_message,
+			"username": config['Slack']['bot_username'],
+			"icon_emoji": config['Slack']['bot_emoji'],})
+		return
```

Comment:
Same. Also, if you aren't going to use the value from `resp` then you should avoid assigning it to avoid confusion.